File: C/vowel_or_consonant.c
#include <stdio.h>
#include <ctype.h>

int main() {
    char ch;
    printf("Enter a character: ");
    if (scanf(" %c", &ch) != 1) return 0;

    ch = (char)tolower((unsigned char)ch);

    if (isalpha((unsigned char)ch) && (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u')) {
        printf("Vowel\n");
    } else {
        printf("Consonant\n");
    }

    return 0;
}

File: Python/vowel_or_consonant.py
def main():
    ch = input("Enter a character: ").strip()
    if not ch:
        return
    c = ch[0].lower()

    if c.isalpha() and c in "aeiou":
        print("Vowel")
    else:
        print("Consonant")


if __name__ == "__main__":
    main()

